docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -21,168 +21,109 @@ helpviewer_keywords:
-- private\_protected
-- local
-
-   The `local` accessibility level applies to symbols defined within a method. It's useful for defining naming rules for symbols whose accessibility can't be specified in code. For example, if you specify `applicable_accessibilities = local` on a naming convention for constants (`required_modifiers = const`), the rule applies only to constants defined within a method and not those defined in a type.
-
-   ```csharp
-   class TypeName
-   {
-     // Constant defined in a type.
-     const int X = 3;
-
-     void Method()
-     {
-       // Constant defined in a method with "local" accessibility.
-       const int Y = 4;
-     }
-   }
-   ```
-
-### Symbol modifiers (optional)
-
-To describe the modifiers of the symbols you want the naming rule to apply to, specify a property name in the following format:
+Each property should only be set once, but some settings allow multiple comma-separated values.
 
-`dotnet_naming_symbols.<symbolTitle>.required_modifiers = <values>`
+The order of the properties is not important.
 
-The following list shows the allowable values (separate multiple values with a comma):
+### \<prefix>
 
-- `abstract` or `must_inherit`
-- `async`
-- `const`
-- `readonly`
-- `static` or `shared`
+**\<prefix>** specifies which kind of element is being defined -- naming rule, symbol group, or naming style -- and must be one of the following:
 
-   > [!NOTE]
-   > If you have a naming rule for `static` or `shared` symbols, it is also applied to `const` symbols because they are implicitly static. If you don't want the `static` naming rule to apply to `const` symbols, create a separate naming rule for `const` symbols.
+| To set a property for | Use the prefix | Example |
+| --- | --- | -- |
+| Symbol group | `dotnet_naming_symbols` | `dotnet_naming_symbols.interface.applicable_kinds = interface` |
+| Naming style | `dotnet_naming_style` | `dotnet_naming_style.pascal_case.capitalization = pascal_case` |
+| Naming rule | `dotnet_naming_rule` | `dotnet_naming_rule.types_should_be_pascal_case.severity = suggestion`
 
-A naming rule matches signatures that have *all* the modifiers specified in `required_modifiers`. If you omit this property, the default value of an empty list is used, that is, no specific modifiers are required for a match. This means a symbol's modifiers have no effect on whether or not this rule is applied.
+Each type of definition -- [naming rule](#naming-rule-properties), [symbol group](#symbol-group-properties), or [naming style](#style-properties)  -- has its own supported properties, as described below.
 
-> [!TIP]
-> Do not specify a value of `*` for `required_modifiers`. Instead, just omit the `required_modifiers` property altogether and your naming rule will apply to any kind of modifier.
+### \<title>
 
-## Style
+**\<title>** is a descriptive name that you choose, which associates multiple property settings into a single definition. For example, the following:
```

Comment:
```suggestion
**\<title>** is a descriptive name that you choose that associates multiple property settings into a single definition. For example, the following properties produce two naming rule definitions, `interface` and `types`, each of which has two properties set on it.
```

Reply:
1. Is there any way to avoid the double _that_ of _a descriptive name that you choose that associates_?
2. I think it important to differentiate between "naming rule definition" and "symbol group definition". I'll address this in a later commit.